test/functional/rpc_createmultisig.py

```diff
@@ -7,9 +7,13 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_raises_rpc_error,
+    assert_equal,
 )
-import decimal
+from test_framework.key import ECPubKey
 
+import binascii
+import decimal
+import itertools
```

Comment:
Unless I'm mistaken, convention is to import system libs first, then own stuff. `decimal` was imported here, though, so grain-of-salt.

Reply:
A bunch of other tests do system libs after our own :man_shrugging: